### ISSUE TITLE
Zoom is broken after update (channel 18.09 AND unstable)

### ISSUE THREAD

**Issue by svalaskevicius** [NixOS/nixpkgs#52498]
## Issue description

Zoom does not start - please see attached screenshots.
![2018-12-18-195848_1716x990](https://user-images.githubusercontent.com/893660/50179888-1644c600-0300-11e9-99fa-dcd9afda3f51.png)

![2018-12-18-195901_3815x675](https://user-images.githubusercontent.com/893660/50179915-26f53c00-0300-11e9-85c3-4c487186afd6.png)

error from the unstable channel is the same (diff zoom version):
```
 ~  zoom-us                                                                                                                                 3m  Tue 18 Dec 2018 20:03:27 GMT
zoom started.
[CZPClientLogMgr::LogClientEnvironment] [MacAddr: E4:70:B8:95:DC:42][client: Linux][OS: NixOS 18.09.1676.7e88992a8c7 (Jellyfish)][Hardware: CPU Core:4 Frenquency:2.9 G Memory size:15897MB CPU Brand:Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz GPU Brand:][Req ID: ]
Linux Client Version is 2.5.146186.1130
QSG_RENDER_LOOP is 
XDG_CURRENT_DESKTOP = ;   GDMSESSION = none+xmonad
Graphics Card Info:: 00:02.0 VGA compatible controller: Intel Corporation Device 591b (rev 04)
Zoom package arch is 64bit, runing OS arch is x86_64
AppIconMgr::systemDesktopName log Desktop Name: none+xmonad 
Error: Send error, 22 Invalid argument
Error: Send error, 22 Invalid argument
Error: Send error, 22 Invalid argument
[16566:36302377:12-18/20:05:05.766:FATAL:SBConfUI.cpp(57)] Assert failed: !g_pConfUI. 
```

### Steps to reproduce

Upgrade to latest 18.09 (also changed to modesetting xorg driver, and set linux kernel to the latest if related)

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.9, NixOS, 18.09.1676.7e88992a8c7 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - channels(sarunas): `""`
 - channels(root): `"nixos-18.09.1676.7e88992a8c7, nixos-unstable-19.03pre161900.61c3169a0e1"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by svalaskevicius** [comment]
update: tried restoring old xorg driver, same result :(

**Comment by svalaskevicius** [comment]
tried running old zoom versions from the nix store, all result in the same screenshot, however the older ones don't have the "Assert failed: !g_pConfUI." line - so I guess it's not the cause

**Comment by svalaskevicius** [comment]
tried with older kernel - the same

**Comment by svalaskevicius** [comment]
hmm, as much as I've tried only booting to the previous generation makes zoom working again - thus, I'm stuck and cannot upgrade until this is resolved

**Comment by svalaskevicius** [comment]
interesting, zoom works on my other laptop, with *exactly* the same nixos configuration. aren't the same configs with the same updates supposed to work exactly the same on nixos? what could be different?

**Comment by svalaskevicius** [comment]
solved one issue - deleting files from ~/.cache/*qml* and ~/.cache/zoom* helped.

now it's unhappy about SSL certificate validation for "Go Daddy Root Certificate Authority - G2"

**Comment by svalaskevicius** [comment]
straced... it tries to connect to 52.202.62.235, which is zoom.
the same process also reads /etc/ssl/certs/ca-certificates.crt - `curl https://asd.zoom.us --resolve 'asd.zoom.us:443:52.202.62.235'` validates fine, which points to an issue within zoom app itself / Qt.

result: closing this ticket as a non issue :)

**Comment by eethann** [comment]
I am also having this issue. Deleting the flies as noted here doesn't seem to fix.

**Comment by svalaskevicius** [comment]
@eethann I see I missed the star after ~/.cache/qml*. please try (re)moving all cache directories that are named after qt, qml, zoom. I had the same issue several times now and this always helps.

Alternatively, you could check with a new user on Linux, that won't have any local caches

**Comment by jameysharp** [comment]
I didn't have anything named `.cache/*qml*`, and just deleting `.cache/zoom*` wasn't enough, but after I removed `.cache/*qt*`, then zoom-us worked for me again. Thanks for the tips, @svalaskevicius! This doesn't seem likely to be a NixOS specific bug, and it isn't clear to me there's any reasonable workaround the NixOS packaging could provide, so I think it makes sense to leave this issue closed, but I'm glad it was filed here where I could find it when I ran into this issue too. :smile:
